projects/plugins/jetpack/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Jetpack",
-	"version": "13.2.0-a.10",
+	"version": "13.2.0-a.12",
```

Comment:
Should this be `13.2.0-a.11`?

Reply:
No, trunk is at even intervals and the released weeklies use odd numbers so trunk isn't seen as older than an older released version.

Reply:
Shows how little I know about the release processes!